index.html

```diff
@@ -24,20 +24,186 @@
+          </div>
+        </div>
+        <!-- ---------------Work 3----------------- -->
+        <div class="grid-item">
+          <div class="card-container">
+            <img
+              src="images/work3.png"
+              alt="My firts portfolio work"
+              class="works-image"
+            />
+            <h2 class="work-title">Tonic</h2>
+            <ul>
+              <li><h3 class="work-client h3-common-styles">CANOPY</h3></li>
+              <li><img src="images/dot.png" alt="Dot icon image" /></li>
+              <li><h3 class="work-role h3-common-styles">Back End Dev</h3></li>
+              <li><img src="images/dot.png" alt="Dot icon image" /></li>
+              <li><h3 class="work-year h3-common-styles">2015</h3></li>
+            </ul>
+            <p class="work-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="works-btns">
+              <li><button type="button" class="works-btn">html</button></li>
+              <li><button type="button" class="works-btn">css</button></li>
+              <li>
+                <button type="button" class="works-btn">javascript</button>
+              </li>
+            </ul>
+            <button class="see-project-btn" type="button">See Project</button>
+          </div>
+        </div>
+        <!-- ---------------Work 4----------------- -->
+        <div class="grid-item">
+          <div class="card-container">
+            <img
+              src="images/work4.png"
+              alt="My firts portfolio work"
+              class="works-image"
+            />
+            <h2 class="work-title">
+              Multi-Post <br />
+              Stories
+            </h2>
             <ul>
+              <li><h3 class="work-client h3-common-styles">CANOPY</h3></li>
+              <li><img src="images/dot.png" alt="Dot icon image" /></li>
+              <li><h3 class="work-role h3-common-styles">Back End Dev</h3></li>
+              <li><img src="images/dot.png" alt="Dot icon image" /></li>
+              <li><h3 class="work-year h3-common-styles">2015</h3></li>
+            </ul>
+            <p class="work-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="works-btns">
+              <li><button type="button" class="works-btn">html</button></li>
+              <li><button type="button" class="works-btn">css</button></li>
+              <li>
+                <button type="button" class="works-btn">javascript</button>
+              </li>
+            </ul>
+            <button class="see-project-btn" type="button">See Project</button>
+          </div>
+        </div>
+      </section>
+      <!-- -----------------------------The About Myself Section--------------------------------- -->
+      <section class="about-myself">
```

Comment:
- [ Required ] Good job working on the `About Myself` section it looks great, but I have just one observation the `margin-left` is not consistent with the Figma template. Please see the screenshot below for more details on this and kindly rectify it.

## Figma template
 ![Screenshot from 2023-01-13 23-00-16](https://user-images.githubusercontent.com/16392046/212409123-ca1db23c-89ac-411e-8ac7-c4138e0ef62c.png)


## Your design
![Screenshot from 2023-01-13 22-53-19](https://user-images.githubusercontent.com/16392046/212408742-cc4cf041-b7c7-4d07-bff0-fba5cfaa204f.png)